frontend/config/dev.webpack.config.js

```diff
@@ -13,6 +13,20 @@ const betaOrStable = process.env.BETA ? 'beta' : 'stable';
 // for accessing prod-beta change this to 'prod-beta'
 const env = `${environment}-${betaOrStable}`;
 
+const pluginProxy = (name, port = 8003, pathRewrite = false) => ({
+  context: [`/beta/api/plugins/${name}`, `/api/plugins/${name}`],
+  // In order to serve your plugin locally on your server change this line ↓
+  target: `http://localhost:${port}`,
+  secure: false,
+  changeOrigin: true,
+  ...(pathRewrite && {
```

Comment:
@karelhala to sum up these changes:
1. we will now by default omit the `pathRewrite` config arg
2. We have changed the default port to `8003` (from `9000`)
3. `context` will now be based on the `name` passed into the config function

Is that correct?

Reply:
Somewhat yes, let me summarize these changes. Because any hac plugin lives on CRC we don't need the `pathRewrite`, that's only for custom plugins (for instance those hosted on openshift console). The port is just the same, when you run hac-dev the port defaults to `8003` instead of the one of openshift console. And the context is autocalculated, if you look at how openshift console backend exposes plugin and how hac-dev is shown both have `/api/plugins/$PLUGIN_NAME`. Should we need some more custom configuration we can always add another proxy config similiar to what is used for `/api`.